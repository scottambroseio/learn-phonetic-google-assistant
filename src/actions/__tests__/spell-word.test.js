// @flow

import spellWord from '../spell-word';

describe('spell-word', () => {
  it('should generate a correct response', () => {
    const appMock = { tell: jest.fn(), getArgument: () => 'cat' };
    const appMock2 = { tell: jest.fn(), getArgument: () => 'dandelion' };

    spellWord(appMock);
    spellWord(appMock2);

    expect(appMock.tell.mock.calls[0][0]).toMatchSnapshot();
    expect(appMock2.tell.mock.calls[0][0]).toMatchSnapshot();
  });
});
