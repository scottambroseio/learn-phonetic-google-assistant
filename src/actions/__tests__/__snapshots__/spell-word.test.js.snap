// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spell-word should generate a correct response 1`] = `"The spelling of cat is charlie alfa tango"`;

exports[`spell-word should generate a correct response 2`] = `"The spelling of dandelion is delta alfa november delta echo lima india oscar november"`;
